packages/infra/src/storage.ts

```diff
@@ -117,7 +117,7 @@ export class Storage extends Construct {
       });
 
       const zone = route53.HostedZone.fromLookup(this, 'Zone', {
-        domainName: config.domainName.split('.').slice(-2).join('.'),
+        domainName: config.domainName,
```

Comment:
This is to get the correct hosted zone. Because our hosted zones on AWS are subdomains, we need more than two `.` characters. It's easiest to use `domainName` directly.